React/Containers.md

```diff
@@ -0,0 +1,31 @@
+## ./src/components/containers/Login.tsx
+
+- fetch should not be used here, only thru appData
+- this is **not a partial** do not use containers as such (e.g. `PaginationContainer`)
+- no styled components here (margins should happen on the components thru the props)
+
+```tsx
+import React from 'react';
+
+import LoginForm from 'components/ui/LoginForm';
+import appData from 'collections/app-data';
```

Comment:
It is not a good idea to import the whole app-data inside this component.
We should pass it down through context.